canvas_account_admin_tools/settings/base.py

```diff
@@ -267,4 +267,6 @@
 PERMISSION_PEOPLE_TOOL = 'people_tool'
 PERMISSION_XLIST_TOOL = 'cross_listing'
 
-
+# in search courses, when you add a person to a course. This list
+# controls which roles show up in the drop down
```

Comment:
Aren't these actually `UserRole` ids? (not Canvas role ids?) Either way, we can be more specific about that in the comment.